### PR TITLE
Cotco 686 complete by reviewer

### PR DESCRIPTION
**Pull request by peppe-hmrc**
This PR is for the reviewer complete only

### CODE REVIEW

File: app/uk/gov/hmrc/ctc/controllers/PortSelectionForReviewController.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/controllers/PortSelectionForReviewController.scala]: This file should be removed. The entry point is in AggregationController

Comment (peppe-hmrc) [reply app/uk/gov/hmrc/ctc/controllers/PortSelectionForReviewController.scala]: yep true

File: app/uk/gov/hmrc/ctc/controllers/actions/Actions.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ import javax.inject.Inject
 import play.api.mvc.{Action, ActionBuilder}
 
 class Actions @Inject()(
-    requestWithProvider: RequestWithProviderAction,
+    requestWithProvider: RequestWithProviderIdAction,
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/controllers/actions/Actions.scala]: Why RequestWithProviderIdAction ? The request is now containing not only the providerId but even the provider's role.

Comment (peppe-hmrc) [reply app/uk/gov/hmrc/ctc/controllers/actions/Actions.scala]: this a merge conflict issue ... Gonna fix it

File: app/uk/gov/hmrc/ctc/controllers/actions/RequestWithProvider.scala [hunk 3]
```diff
@@ -21,17 +21,15 @@ import scala.concurrent.{ExecutionContext, Future}
...
 import play.api.mvc.Results.BadRequest
-import uk.gov.hmrc.ctc.models.api.Headers
+import uk.gov.hmrc.ctc.models.api.{Headers, Roles}
 import uk.gov.hmrc.ctc.models.types.ModelTypes.ProviderId
 import uk.gov.hmrc.play.HeaderCarrierConverter
 
 case class RequestWithProvider[A](isReviewer: Boolean, providerId: ProviderId, request: Request[A]) extends WrappedRequest[A](request)
 
-class RequestWithProviderAction @Inject()()(implicit executionContext: ExecutionContext)
+class RequestWithProviderIdAction @Inject()()(implicit executionContext: ExecutionContext)
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/controllers/actions/RequestWithProvider.scala]: Why RequestWithProviderIdAction ? The request is now containing not only the providerId but even the provider's role.

File: app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewData.scala [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewData.scala]: This file should be removed. The right file is `RejectedPerPortData.scala`

File: app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewPageData.scala [hunk 5]
```diff
@@ -0,0 +1,28 @@
+/*
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewPageData.scala]: This file should be removed. The right file is `RejectedPerPortAggregate.scala`.

File: app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewData.scala (hunk 4 above)
Comment (peppe-hmrc) [reply app/uk/gov/hmrc/ctc/models/api/PortSelectionForReviewData.scala]: I don't understand why these files are still here ... they should have disappeared after the rebase, isn't it? Anyway, going to delete them

File: app/uk/gov/hmrc/ctc/services/OfficeOfTransitService.scala [hunk 6]
```diff
@@ -35,69 +41,79 @@ class OfficeOfTransitService @Inject()(
...
+
+  def completeUnProcessed(
       transitId: TransitId,
       officeOfTransit: OfficeOfTransit,
       userType: UserType,
       providerId: ProviderId): ServiceResponse[TransitId] = {
 
     val operatorId = s"$userType:$providerId"
     val processingResult: ProcessingResult = ProcessingResult(userType, providerId, officeOfTransit)
+    val isCompleted: Boolean = isActuallyCompleted(processingResult.officeOfTransit)
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/services/OfficeOfTransitService.scala]: Why is not this check evaluated only for `case Right(...` ?
It's not necessary for `case Left(...`

File: app/uk/gov/hmrc/ctc/services/PortSelectionForReviewService.scala [hunk 7]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (udinobi) [inline app/uk/gov/hmrc/ctc/services/PortSelectionForReviewService.scala]: This file should be removed. The right file is `RejectedPerPortService.scala`.

File: it/uk/gov/hmrc/ctc/TransitControllerISpec.scala [hunk 8]
```diff
@@ -133,7 +133,7 @@ class TransitControllerISpec extends BaseISpec {
     }
   }
 
-  "GET /transits" should {
+  "GET /transits/rejected" should {
```
Comment (udinobi) [inline it/uk/gov/hmrc/ctc/TransitControllerISpec.scala]: Uhu? Where this endpoint is coming from ?

**Pull request merged**